09-redux/classworks/Malkov_Aleksei/src/components/services/fetcherAbout.js

```diff
@@ -0,0 +1,7 @@
+async function fetcherAbout(fetchReddit, value) {
```

Comment:
there is no need to create separate files for every action. use `koa-react-starter` as a reference. we have `resources` folder with folders for every entity inside. and every entity folder has `entity.actions.js` file